ClosedXML/Excel/Tables/XLTable.cs

```diff
@@ -860,5 +865,369 @@ internal IXLTable CopyTo(XLWorksheet targetSheet, bool copyData = true)
             }
             return newTable;
         }
+
+        #region Append and replace data
+
+        public IXLRange AppendData(IEnumerable data)
+        {
+            return AppendData(data, false);
+        }
+
+        public IXLRange AppendData(IEnumerable data, bool transpose)
```

Comment:
I've noticed this too, but haven't spent much time on it. This issue originated in `XLCell.InsertData` and `XLCell.InsertTable`. As far as I could gather the non-generic version would allow objects of different types to be dumped to the worksheet. The generic version restricts objects to the same time, and also has better performance because of this.

But I will have another look at it.

Reply:
Actually, transposing doesn't really make sense for a table. I got this from the `XLCell.InsertData` and there it makes sense if e.g. you have an enumerable of simple types, e.g. an array of strings and you want to dump it into a table. I don't think we should allow it for 2-dimensional structures.

Reply:
Just to clarify, when no transposing, if you do `XLCell.InsertData(new [] {"a", "b", "c"})`, then the items will be set vertically, under each other. I added the `transpose` parameter a while back to enable users to dump the values in a single row, horizontally.